rasa/nlu/tokenizers/tokenizer.py

```diff
@@ -28,19 +28,19 @@ def set(self, prop: Text, info: Any) -> None:
     def get(self, prop: Text, default: Optional[Any] = None) -> Any:
         return self.data.get(prop, default)
 
-    def __eq__(self, other) -> bool:
+    def __eq__(self, other):
         if not isinstance(other, Token):
-            return NotImplemented
```

Comment:
this should stay `NotImplemented` https://docs.python.org/3/library/constants.html#NotImplemented